pkg/controllers/edgeconnect/consts/consts.go

```diff
@@ -23,4 +22,7 @@ const (
 	KeyEdgeConnectId                = "id"
 
 	AnnotationEdgeConnectContainerAppArmor = "container.apparmor.security.beta.kubernetes.io/" + EdgeConnectContainerName
+
+	// consts.SecretConfigConditionType identifies the secret config condition.
+	SecretConfigConditionType = "consts.SecretConfigConditionType"
```

Comment:
the `consts.` in the type doesn't really serve any purpose IMO

Reply:
lol it was by mistake :)